### PR TITLE
Add timestamps in Log model

### LINKED ISSUE #13: Add timestamps in Log model
Add a new field in Log model name `created_at`. This will be a DateTimeField with default value as `now()`

TODOs
- [ ] Add the `created_at` field in the Log model with default value as now
- [ ] Make migrations
- [ ] Test if the Logs are generated with a correct timestamp using the sendrequest command.
    - [ ] Create a sample job in admin panel
    - [ ] Run the job
    ```
    python manage.py sendrequest --job_id=1
    ```

### PR DESCRIPTION
**Pull request by prateek2211**
fix #13 
* [X] Add the created_at field in the Log model with default value as now
* [X] Make migrations
* [X] Test if the Logs are generated with a correct timestamp using the sendrequest command.
     * [X] Create a sample job in admin panel
     * [X] Run the job

### CODE REVIEW

File: cron/models.py [hunk 1]
```diff
@@ -33,11 +35,13 @@ def fullURL(self):
   def __str__(self):
     return self.name
 
+
 class Log(models.Model):
   id = models.AutoField(primary_key=True)
   job = models.ForeignKey(Job, on_delete=models.CASCADE)
   statusCode = models.IntegerField()
   response = models.TextField()
+  created_at = models.DateTimeField(default=timezone.now)
```
Comment (jatinkatyal13) [inline cron/models.py]: use https://docs.djangoproject.com/en/2.2/ref/models/fields/#django.db.models.DateField.auto_now_add

File: poker/settings.py [hunk 2]
```diff
@@ -94,7 +94,7 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = 'Asia/Kolkata'
```
Comment (jatinkatyal13) [inline poker/settings.py]: need not change time zone

File: cron/models.py (hunk 1 above)
Comment (prateek2211) [reply cron/models.py]: done

File: cron/models.py [hunk 3]
```diff
@@ -1,6 +1,8 @@
 from django.db import models
 import urllib.parse
 
+from django.utils import timezone
```
Comment (jatinkatyal13) [inline cron/models.py]: Remove this import and it's a good to go

Comment (prateek2211) [reply cron/models.py]: ok

Comment (prateek2211) [inline cron/models.py]:
```suggestion

```

Comment (prateek2211) [reply cron/models.py]: done

**Review by jatinkatyal13 — APPROVED** [review]

**Pull request merged**